src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.ts

```diff
@@ -604,6 +615,23 @@ export class InstructorSessionsPageComponent extends InstructorSessionModalPageC
       }, (resp: ErrorMessageOutput) => {
         this.statusMessageService.showErrorToast(resp.error.message);
       });
-    }, () => {});
+    });
+  }
+
+  /**
+   * Attempts to load all data again.
+   */
+  retryLoadingAllData(): void {
+    this.hasFeedbackSessionLoadingFailed = false;
+    this.hasCourseLoadingFailed = false;
+    this.loadCandidatesCourse();
```

Comment:
Again, I think it is `loadCandidatesCourse`'s responsibility to set `hasCourseLoadingFailed` and `courseCandidates` at the begining.